themes/README.md

```diff
@@ -1,6 +1,43 @@
-## Gatsby Themes and Starters
+<p align="center">
```

Comment:
Please remove the whole header as this is not a public facing README and it creates a bit of churn.

The old header works fine.

```
# Gatsby Themes and Starters

Here are the first party themes maintained by Gatsby.
```

Reply:
I have done the required changes in the recent commit